index.html

```diff
@@ -119,7 +122,10 @@ <h2>Preserving Tomorrow by Building Today</h2>
 					<div class="community-card">
 						<!-- img -->
 						<div class="img-container">
-							<img src="https://placedog.net/800/640?id=7" alt="doggo" />
+							<img
+								src="./assets/images/pages/home/community-partnership.jpg"
```

Comment:
@maxharrisnet can you please crop and reoptimize this picture? it has a rounded corner baked into the file.